src/omd.ml

```diff
@@ -16,7 +16,10 @@ let parse_inlines (md, defs) =
 
 let of_channel ic = parse_inlines (Pre.of_channel ic)
 let of_string s = parse_inlines (Pre.of_string s)
-let to_html doc = Html.to_string (Html.of_doc doc)
+
+let to_html ?auto_identifiers doc =
+  Html.to_string (Html.of_doc ?auto_identifiers doc)
```

Comment:
In order to run our tests with this flag set to false, I think we can just set the `auto_identifiers` flag to `false` in the call to `to_html` here:

https://github.com/ocaml/omd/blob/191414e6525a811803dd5285dc07848882859c88/tests/omd.ml#L20-L22